app/controllers/application_controller.rb

```diff
@@ -45,4 +45,14 @@ def ssl_required?
   def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: [:name, :organisation])
   end
+
+  # this method is to respond to the will_paginate bug of invalid page number leading to error being thrown.
+  # see discussion here https://github.com/mislav/will_paginate/issues/271
+  def validate_page_param
+    unless params[:page].present? && params[:page].to_i > 0
```

Comment:
[Do not use unless with else. Rewrite these with the positive case first.](https://github.com/bbatsov/ruby-style-guide#no-else-with-unless)

I also notice your indentation is a little out on the params lines - you need an extra space.